core/labeled-replies-3.3.md

```diff
@@ -40,7 +40,7 @@ Additionally, labeled responses allow bouncers with multiple connected clients t
 
 ### Dependencies
 
-This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and relies on support for the [`batch`](/specs/extensions/batch-3.2.html) capability which MUST be negotiated at the same time.
+This specification uses the [message tags framework](/specs/core/message-tags-3.2.html) and uses the [`batch`](/specs/extensions/batch-3.2.html) capability which SHOULD be negotiated at the same time.
```

Comment:
No probs, I find it better to comment on the "files changed" view rather than a specific commit.

SHOULD means exactly that :) Strongly recommended where it makes sense. It's a stronger recommendation for server than clients really, but clients will have a better time if they enable it too.